test/sql/copy/csv/full_file_download_bug.test

```diff
@@ -0,0 +1,12 @@
+# name: test/sql/copy/csv/full_file_download_bug.test
```

Comment:
I would probably remove this test since the content-length has since been added to that file, so that it can work on v0.9.2